Operational-Procedures/Scheduler/index.rst

```diff
@@ -0,0 +1,72 @@
+.. _scheduler-operational-procedures:
+
+################################
+Scheduler Operational Procedures
+################################
+
+The Scheduler CSC is in charge of computing an observing queue for the observatory.
+There are two instances of the CSC running at all times, one dedicated to the Main Telescope and another to the Auxiliary Telescope, with indexes 1 and 2 respectively.
+
+Both CSCs share the exact same code-base, which means operating and throubleshooting shares several commonalities.
+
+This page focus on the operational aspects of the Scheduler CSC, in general.
+Any difference between the operation of MT and AT Scheduler will be appropriately highlighted. 
+
+For more details about the CSC, see the `Scheduler CSC documentation`_.
+
+.. _Scheduler CSC documentation: https://ts-scheduler.lsst.io
+
+If you are interested in the official scheduling algorithm, that is used to compute the observing queue, see the `Feature Based Scheduler documentation`_.
+
+.. _Feature Based Scheduler documentation: https://rubin-sim.lsst.io
+
+
+.. _scheduler-operational-procedures-overview:
+
+Overview
+========
+
+Before diving in the operational procedures, let's have a quick overview of how the Scheduler CSC operates.
+
+The Scheduler CSC can be divided into two main domains; *observatory interaction* and the *scheduling algorithm*.
```

Comment:
are these components? "Domains" feels a bit odd... 
Something like, 
"The Scheduler CSC can be divided into two main components; An *observatory interaction component* and the *scheduling algorithm component*.